### ISSUE TITLE
Data augument for supervised models

### ISSUE THREAD

**Issue by Dashi-1997** [google-research/simclr#110]
Hello, does the supervised comparison model you provided here use the same data enhancement method（random crop (with resize and random flip), random color distortion, and random Gaussian blur）?
![image](https://user-images.githubusercontent.com/64891953/105499343-4b504d80-5cfc-11eb-85c4-44b959da6368.png)

**Comment by chentingpc** [comment]
No, they are trained with standard resnet settings (e.g. random crop augmenttion, no color augmentation or gaussian blur). With the extra augmentations and a longer training schedule, you could expect the supervised models to be improved roughly 0.5-2%.

**Comment by Zoe-Wan** [comment]
Hello, does the supervised comparison model  resnet50 1x use the same hyperparameter, optimizer and the scheduler as the example settings as follows? 
( batch_size: 256, baselr: 0.1, weight_decay: 1e-4, optimizer: SGD(momentum: 0.99), scheduler: StepLR(step_size=30, gamma:0.1)
Using the settings above, I can't reproduce your acc.  😢

**Comment by Zoe-Wan** [comment]
Hello @chentingpc , does the supervised comparison model resnet50 1x use the same hyperparameter, optimizer and the scheduler as the example settings as follows?
( batch_size: 256, baselr: 0.1, weight_decay: 1e-4, optimizer: SGD(momentum: 0.99), scheduler: StepLR(step_size=30, gamma:0.1)
Using the settings above, I can't reproduce your acc. 😢
